setup.py

```diff
@@ -16,7 +16,7 @@
 
 setup_args = {
     "name": "ndx-miniscope",
-    "version": "0.3.0",
+    "version": "0.3.1",
```

Comment:
This is bigger than a patch
```suggestion
    "version": "0.4.0",
```